spatialprofilingtoolbox/apiserver/app/main.py

```diff
@@ -1,23 +1,36 @@
 """The API service's endpoint handlers."""
-import os
 import json
-import re
 from io import BytesIO
-from base64 import b64encode
 from base64 import b64decode
 
-from PIL import Image
 from fastapi import FastAPI
+from fastapi.openapi.utils import get_openapi
 from fastapi import Query
 from fastapi import Response
 from fastapi.responses import StreamingResponse
 
-from spatialprofilingtoolbox.db.fractions_transcriber import \
-    describe_fractions_feature_derivation_method
-from spatialprofilingtoolbox.apiserver.app.db_accessor import DBAccessor
 from spatialprofilingtoolbox.ondemand.counts_service_client import CountRequester
-from spatialprofilingtoolbox.workflow.common.export_features import ADIFeatureSpecificationUploader
-VERSION = '0.4.0'
+from spatialprofilingtoolbox.db.exchange_data_formats.study import StudyHandle
+from spatialprofilingtoolbox.db.exchange_data_formats.study import StudySummary
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import CellFractionsSummary
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import PhenotypeSymbol
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import PhenotypeCriteria
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import PhenotypeCounts
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import \
+    ProximityMetricsComputationResult
+from spatialprofilingtoolbox.db.exchange_data_formats.metrics import UMAPChannel
+from spatialprofilingtoolbox.db.querying import query
+from spatialprofilingtoolbox.apiserver.app.validation import ValidChannel
+from spatialprofilingtoolbox.apiserver.app.validation import ValidStudy
+from spatialprofilingtoolbox.apiserver.app.validation import ValidPhenotypeSymbol
+from spatialprofilingtoolbox.apiserver.app.validation import ValidPhenotype1
+from spatialprofilingtoolbox.apiserver.app.validation import ValidPhenotype2
+from spatialprofilingtoolbox.apiserver.app.validation import ValidChannelListPositives
+from spatialprofilingtoolbox.apiserver.app.validation import ValidChannelListNegatives
```

Comment:
This is a writer's choice thing, but I am curious why you choose to do imports this way instead of
```
from spatialprofilingtoolbox.apiserver.app.validation import ValidChannel, ValidStudy, ...
```

Reply:
Oh yeah, I do this so that it is easier to scan the imported symbols visually, and remove one and place it elsewhere during refactoring. Also the linter complains about long lines.

Reply:
One way that I have avoided this repetitious appearance in other source files is by aggregating logical groups of symbols into a class. This could even work for symbols of classes, with a little class-provider class. E.g. here could have `ValidTypes` with `ValidTypes.channel()`, `ValidTypes.study()` etc.

Reply:
I tried to combine them but it doesn't quite work because these symbols need to show up as type hints, and expressions aren't allowed there. I tried a named tuples variant, a class member variant, everything seems to cause a type checking error. So for now we'll just have this big list of `Valid...`.

Reply:
I would write it as
```
from spatialprofilingtoolbox.apiserver.app.validation import \
    (ValidChannel,ValidStudy,ValidPhenotypeSymbol,ValidPhenotype1,ValidPhenotype2,
     ValidChannelListPositives,ValidChannelListNegatives)
```
The slash isn't necessary if you start the tuple on the same line.

Reply:
The thing I don't like about this form is that the scanning task is much harder. E.g.,
*Does this list contain `ValidPhenotypeSymbols`?*
You will no doubt have noticed that in several places, I forced in newlines in enumerations of list items, whether in lists, tuples, argument lists, etc., and this is the reason I do that.

Except for this side effect, it is of course better to do it the above way to save lines and omit needless repetition of the module name. A possible compromise the keeps scannability and reduces module repetition:
```py
from spatialprofilingtoolbox.apiserver.app.validation import (
    ValidChannel,
    ValidStudy,
    ValidPhenotypeSymbol,
    ValidPhenotype1,
    ValidPhenotype2,
    ValidChannelListPositives,
    ValidChannelListNegatives,
)
```
(The final newline before closing parenthesis is my go-to style to make each line item have the same format and to announce the scope level delimited by the closing paren. I use this consistently everywhere I think.)